docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -128,22 +129,22 @@ indicates that there are no more items.
 
 There are two other C# syntax elements that may be new to you. The `using` 
 statement in this method manages resource cleanup. The variable that is
-initialized in the using statement (`reader`, in this example) must
+initialized in the `using` statement (`reader`, in this example) must
 implement the `IDisposable` interface. The `IDisposable` interface
-defines a single method, `Dispose()`, that should be called when the
+defines a single method, `Dispose`, that should be called when the
 resource should be released. The compiler generates that call when
 execution reaches the closing brace of the `using` statement. The
 compiler-generated code ensures that the resource is released even if an
 exception is thrown from the code in the block defined by the using
 statement.
 
-The reader variable is defined using the `var` keyword. `var` defines an
+The `reader` variable is defined using the `var` keyword. `var` defines an
 *implicitly typed local variable*. That means the type of the variable is
 determined by the compile time type of the object assigned to the
-variable. Here, that is the return value from `File.OpenText()`, which is
+variable. Here, that is the return value from `File.OpenText`, which is
 a `StreamReader` object.
```

Comment:
Here again, `FIle.OpenText` could be [File.OpenText](xref:System.IO.File.OpenText(System.String)) so it resolves as a link. Similarly, @System.IO.StreamReader instead of `StreamReader`.